pylot.py

```diff
@@ -165,7 +178,13 @@ def driver():
                                                   point_cloud_stream,
                                                   ground_segmented_stream,
                                                   imu_stream, pose_stream)
-    erdos.run()
+    erdos.run_async()
+
+    # If we did not use the pseudo-asynchronous mode, ask the sensors to
```

Comment:
Is this needed? I don't think simulator_in_sync is used in any non pseudo-asynchronous mode.

Reply:
I think so. In non pseudo-asynchronous mode, we would not have `PlanningPoseSynchronizerOperator` and thus, noone to tell the sensor streams to release data. This line just forces the sensors to release all the data as soon as it is available.